### PR TITLE
Export GOB entities and relations to another database

### PR DESCRIPTION
**Pull request by Vegter**
A request has to be posted with the database configuration.
The configuration is used to populate another database with GOB data.
During the process logging is streamed to be able to follow the progress.

Internally a \COPY like command is used.
However no physical files are used. All information is streamed from the GOB database
to the other database.
Large datasets are divided in multiple COPY's to prevent db overload problems and to be able to
continue streaming progress

### CODE REVIEW

File: src/gobapi/dump/config.py [hunk 1]
```diff
@@ -49,25 +77,52 @@ def get_reference_fields(spec):
...
+    :param field_name:
     :param specs:
     :return:
     """
-    entity_id_field = specs[UNIQUE_ID]['entity_id']
-    volgnummer_field = FIELD.SEQNR
-    id = get_field_value(entity, entity_id_field, specs[entity_id_field])
-    has_volgnummer = specs.get(volgnummer_field) is not None
-    if has_volgnummer:
+    if field_name == UNIQUE_ID:
```
Comment (barykine) [inline src/gobapi/dump/config.py]: not using is_unique_id() here?

Comment (barykine) [reply src/gobapi/dump/config.py]: ah, I see it. nevermind.

File: src/gobapi/api.py [hunk 2]
```diff
@@ -167,16 +171,25 @@ def _dump(catalog_name, collection_name):
...
+        entities, model = dump_entities(catalog_name, collection_name)
 
-    if format == "csv":
-        return Response(csv_entities(entities, model), mimetype='text/csv')
-    elif format == "sql":
-        return Response(sql_entities(catalog_name, collection_name, model), mimetype='application/sql')
-    else:
-        return f"Unrecognised format parameter '{format}'" if format else "Format parameter not set", 400
+        if format == "csv":
+            return Response(csv_entities(entities, model), mimetype='text/plain')
```
[the remark this answers is not in the document]
Comment (Vegter) [reply src/gobapi/api.py]: Good point! Missed that one

File: src/gobapi/api.py [hunk 3]
```diff
@@ -167,16 +171,25 @@ def _dump(catalog_name, collection_name):
...
+        elif format == "sql":
+            return Response(sql_entities(catalog_name, collection_name, model), mimetype='text/plain')
```
[the remark this answers is not in the document]
Comment (Vegter) [reply src/gobapi/api.py]: also here

**Pull request merged**